### PR TITLE
Add beam amanger docs

### PR DESCRIPTION
**Pull request by iwko**

### CODE REVIEW

File: biggerquery/beam_manager.py [hunk 1]
```diff
@@ -164,8 +164,8 @@ def create_dataflow_manager(
         dataflow_bucket,
         requirements_file_path,
         region=DEFAULT_REGION,
-        machine_type='n1-standard-1',
-        internal_tables={},
+        machine_type=DEFAULT_MACHINE_TYPE,
+        internal_tables=None,
         external_tables={},
```
Comment (beczkowb) [inline biggerquery/beam_manager.py]: None instead of {}

File: README.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # BiggerQuery &mdash; Python library for BigQuery
 
 BiggerQuery is a Python library which simplifies working with BigQuery datasets. It wraps BigQuery client, providing elegant
- API for most common use cases.
+ API for most common use cases. It also provides API for creating dataflow pipelines and reading/writing
```
Comment (beczkowb) [inline README.md]: Check the whole readme using grammarly please, there are multiple places missing "the", "a", "an".

File: README.md [hunk 3]
```diff
@@ -468,4 +469,138 @@ class CalculateUserTransactionMetricsTestCase(TestCase):
 
 if __name__ == '__main__':
     main()
-```
\ No newline at end of file
+```
+
+### Creating beam manager
+Beam manager is an object that allows you to create dataflow pipelines using `create_dataflow_pipeline` method and
```
Comment (beczkowb) [inline README.md]: A beam manager object allows you to create a dataflow pipeline. The `create_dataflow_pipeline` function allows you to create the beam manager object. Beam manager provides utility methods, wrapping raw beam API: `write_truncate_to_big_query`, `write_to_avro`, `read_from_big_query`, `read_from_avro`.

File: README.md [hunk 4]
```diff
@@ -468,4 +469,138 @@ class CalculateUserTransactionMetricsTestCase(TestCase):
...
+external_tables = {
+    'transactions': 'dataset.id.transactions',
+    'some_external_table': 'dataset.id2.external_table'
+}
+```
+
+Parameter `runtime` is used to determine partition being processed.
+Parameter `dataflow_bucket` is  GCS bucket used for temporary and staging locations.
+Parameter `requirements_file_path` provides informations about dependencies of your dataflow.
+Parameter `region` is location of machine used in dataflow. By default is set to europe-west1.
```
Comment (beczkowb) [inline README.md]: Is the location of the data center used to process your pipelines.

something like that :P

File: README.md [hunk 5]
```diff
@@ -468,4 +469,138 @@ class CalculateUserTransactionMetricsTestCase(TestCase):
...
+Parameter `machine_type` is type of used machine. By default n1-standard-1.
```
Comment (beczkowb) [inline README.md]: Put some gcp docs link to read more about available machines and the topic in general

File: biggerquery/beam_manager.py (hunk 1 above)
Comment (iwko) [reply biggerquery/beam_manager.py]: Done

File: README.md (hunk 2 above)
Comment (iwko) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (iwko) [reply README.md]: Basing on your suggestion I've made changes

File: README.md (hunk 4 above)
Comment (iwko) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (iwko) [reply README.md]: Done

**Pull request merged**